lib/driver.js

```diff
@@ -497,6 +508,10 @@ class IosDriver extends BaseDriver {
       }
       logger.debug('Waiting for initial webview');
       await this.navToInitialWebview();
+      // we don't need UIAutoClient and Instruments any more,
+      // should stop them to avoid 'Abnormal Instruments error' due to the command timeout.
+      await this.stopUIAutoClient();
```

Comment:
You may be right about that. However, the cost is too high. With automating safari on a simulator, we have access to the full UI of Safari via UIAutomation. If we stop the uiauto client we will no longer have that access. It's very important that we keep it. So I would rather risk a bug that only pops up after an hour of running an app continuously than remove functionality which many people currently rely on.